superdesk/io/registry.py

```diff
@@ -161,10 +162,21 @@ def service(service_id):
             if restricted_parsers is not None:
                 restricted_parsers = list(restricted_parsers.keys())
 
+            fields = getattr(feeding_service_class, 'fields', [])
+            # we need to transtype LazyString to str before serialization
+            for idx, field in enumerate(fields):
+                copied = False
+                for key, value in field.items():
+                    if isinstance(value, LazyString):
```

Comment:
this can probably be avoided by using a `default=str` in the JSON encoder.

Reply:
it should be handled now by [SuperdeskJSONEncoder](https://github.com/superdesk/superdesk-core/blob/develop/superdesk/json_utils.py#L15)